cmds/modprobe/modprobe.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// modprobe - Add and remove modules from the Linux Kernel
+//
+// Synopsis:
+//     modprobe modulename...
```

Comment:
`modprobe modulename [parameters...]`

Reply:
yes, changed that pretty late and did not update...